modules/deploy/pages/deployment-option/cloud/whats-new-cloud.adoc

```diff
@@ -7,9 +7,27 @@ This page lists new features added in Redpanda Cloud.
 
 == May 2024
 
+=== Cloud API: beta
+
+The Cloud API allows you to programmatically manage clusters and resources in your Redpanda Cloud organization. For more information, see the xref:deploy:deployment-option/cloud/api/cloud-api-quickstart.adoc[Cloud API Quickstart], the xref:deploy:deployment-option/cloud/api/cloud-api-overview.adoc[Cloud API Overview], and the full xref:api:ROOT:cloud-api.adoc[Cloud API reference].
+
+=== Redpanda Connect 
```

Comment:
Redpanda Connect is not in Cloud yet so please remove this

Reply:
removed it!